### PR TITLE
add # values to eventIds for logging

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

**Review by henrik-me — APPROVED** [review]
:shipit:

File: src/Microsoft.Identity.Web/LoggingEventId.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// EventIds for Logging.
+    /// </summary>
+    internal static class LoggingEventId
+    {
+        // DistributedCacheAdapter EventIds 100+
```
Comment (henrik-me) [inline src/Microsoft.Identity.Web/LoggingEventId.cs:13]: > // DistributedCacheAdapter EventIds 100+ [](start = 7, length = 41)

nit: the other side of the #pragma

**Review by jmprieur — APPROVED** [review]
LGTM.
Thanks @jennyf19

File: src/Microsoft.Identity.Web/LoggingEventId.cs (hunk 1 above)
Comment (jennyf19) [reply src/Microsoft.Identity.Web/LoggingEventId.cs]: i'm not sure what you are saying, but my cat has an idea.

**Pull request merged**